### DISCUSSION TITLE
New translator for Portuguese (Portugal)

### DISCUSSION THREAD

**Discussion by cong862023** [sircharlo/meeting-media-manager#2005] [Translations]
### What language do you speak?

I would like to help translate M³ into a language I speak, Portuguese (Portugal) (TPO/pt-pt).
For some time, I would like to help with the translation work if it's possible.

**Comment by mtdvlpr** [comment]
@cong862023, thanks for reaching out! Please create an account (using GitHub) at [Crowdin](https://crowdin.com) and go to [this page](https://crowdin.com/project/meeting-media-manager/pt-PT) to request access to our translation project.
